README.md

```diff
@@ -26,11 +26,12 @@ input {rethinkdb
     auth_key => ""
     watch_dbs => ["db1", "db2"]
     watch_tables => ["test.foo", "db2.baz"]
+    backfill => true
     }}
 output {stdout {codec => json_lines}}'
 ```
 
-This will immediately watch the tables `test.foo` and `db2.baz`, and it will also watch the databases `db1` and `db2` for new or dropped tables and watch or unwatch those tables appropriately.
+This will immediately watch the tables `test.foo` and `db2.baz`, and it will also watch the databases `db1` and `db2` for new or dropped tables and watch or unwatch those tables appropriately. Since `backfill` is `true`, it will automatically watch any tables it finds in `db1` or `db2`, and will send events over logstash for each document already in those tables.
```

Comment:
That sentence is a bit redundant. The previous sentence already explains that all tables in `db1` and `db2` are watched. Alternative suggestion:

Since `backfill` is `true`, it will automatically send events for the documents that already exist in those tables over logstash during intialization.